packages/react/src/alert-dialog/trigger/AlertDialogTrigger.tsx

```diff
@@ -39,6 +41,13 @@ export const AlertDialogTrigger = React.forwardRef(function AlertDialogTrigger(
     native: nativeButton,
   });
 
+  const setTriggerElement = React.useCallback(
+    (node: HTMLButtonElement | null) => {
+      store.set('triggerElement', node);
+    },
+    [store],
+  );
+
```

Comment:
This seems to be a common pattern, we might want to add it as a `Store` method at some point.